server.js

```diff
@@ -1,14 +1,42 @@
 require("express-async-errors");
 const express = require("express");
 const path = require("path");
+const passport = require('passport');
+const { Strategy } = require('passport-twitter');
+const { TWITTER_CONSUMER_KEY, TWITTER_CONSUMER_SECRET, SESSION_SECRET } =  process.env;
 const app = express();
 
+const port = 2020 || process.env.PORT;
 const preMiddlewares = require("./src/middlewares/preMiddlewares");
 const errorMiddlewares = require("./src/middlewares/errorMiddlewares");
 const routes = require("./src/routes");
 
 preMiddlewares(app);
 
+passport.use(new Strategy({ 
```

Comment:
can you move this logic to preMiddlewares

Reply:
Done.

Reply:
@usmansbk